m-queue.js

```diff
@@ -201,16 +234,10 @@ module.exports = function(RED) {
                     }
                 } else {
                     qIndex = findQueue(qSelect,qArray)
-                    }        
```

Comment:
This one was really annoying, I had to check your previous commits and I think it should not be there...